### PR TITLE
Main activity view implementation

### PR DESCRIPTION
**Pull request by FranciscoGaraventa**
image

Structure of package:
image

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -1,18 +1,155 @@
...
+        android:id="@+id/text_result"
+        style="@style/TextViewStyle"
+        android:text="@string/text_msj"
+        app:layout_constraintBottom_toTopOf="@id/button_numb_7"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
+    <Button
+        android:id="@+id/button_numb_7"
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: do not use numbers in the IDs pls

File: app/src/main/res/values/dimensions.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="margin_button_xsmall_10">10dp</dimen>
```
Comment (SartiMau) [inline app/src/main/res/values/dimensions.xml]: same here, let the names without numbers

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,3 +1,23 @@
 <resources>
     <string name="app_name">Calculator</string>
+    <string name="button_number_0">0</string>
```
Comment (SartiMau) [inline app/src/main/res/values/strings.xml]: same

File: app/src/main/res/values/styles.xml [hunk 4]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="ButtonStyle">
+        <item name="android:layout_width">@dimen/width_button_xbig_60</item>
```
Comment (SartiMau) [inline app/src/main/res/values/styles.xml]: as a standard (and good practice) the **android:layout_width** and the **android:layout_height** should be in the XML, not in a style

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_main.xml [hunk 5]
```diff
@@ -1,18 +1,155 @@
...
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="Hello World!"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
+        android:id="@+id/text_result"
+        style="@style/TextViewStyle"
+        android:text="@string/text_msj"
+        app:layout_constraintBottom_toTopOf="@id/button_numb_7"
```
Comment (MarcosCGlobant) [inline app/src/main/res/layout/activity_main.xml]: There is no need to abbreviate strings or ids, i recommend using the full words in a concise manner.
Check in the PR where to do this changes, there is possible in multiple places.

File: app/src/main/res/values/colors.xml [hunk 6]
```diff
@@ -7,4 +7,7 @@
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+    <color name="colorBoulder">#787575</color>
+    <color name="colorFlushMahogany">#CF473A</color>
+    <color name="colorScorpion">#575757</color>
```
Comment (MarcosCGlobant) [inline app/src/main/res/values/colors.xml]: You can use descriptive names on the colours, related to the place where they are used to keep consistency
```suggestion
    <color name="button_background_tint">#787575</color>
    <color name="op_button_background_tint">#CF473A</color>
```

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (FranciscoGaraventa) [reply app/src/main/res/layout/activity_main.xml]: Numbers in ids were changed to name of the number.

File: app/src/main/res/values/dimensions.xml (hunk 2 above)
Comment (FranciscoGaraventa) [reply app/src/main/res/values/dimensions.xml]: Done

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (FranciscoGaraventa) [reply app/src/main/res/values/strings.xml]: Done

File: app/src/main/res/values/styles.xml (hunk 4 above)
Comment (FranciscoGaraventa) [reply app/src/main/res/values/styles.xml]: Added layout_width and layout_height in activity_main and removed from styles.

File: app/src/main/res/layout/activity_main.xml (hunk 5 above)
Comment (FranciscoGaraventa) [reply app/src/main/res/layout/activity_main.xml]: Done

File: app/src/main/res/values/colors.xml (hunk 6 above)
Comment (FranciscoGaraventa) [reply app/src/main/res/values/colors.xml]: Color constant names have been changed to be more descriptive.

**Review by SartiMau — DISMISSED** [review]

**Review by MarcosCGlobant — APPROVED** [review]

**Review by SartiMau — APPROVED** [review]

**Pull request merged**